website/src/pages/documents/document.page.tsx

```diff
@@ -82,6 +83,10 @@ const TabSet = ({ doc }: { doc: Document }) => {
           </Tab>
         </TabList>
       </div>
+      <div id="scroll-top" className={css.scrollTop} onClick={() => window.scrollTo({top: 0, behavior: 'smooth'})}>
```

Comment:
Agree with Naomi, also consider changing this `div` to a `button`, because it has the function of a button!